ngsderive/commands/strandedness.py

```diff
@@ -116,10 +118,9 @@ def determine_strandedness(
     if "RG" in samfile.header:
         read_groups += [rg["ID"] for rg in samfile.header["RG"]]
 
-    overall_evidence = defaultdict(dict)
-
     for rg in read_groups:
-        overall_evidence[rg] = defaultdict(int)
+        if not overall_evidence[rg]:
+            overall_evidence[rg] = defaultdict(int)
```

Comment:
Instead of doing this, could we do

```suggestion
            overall_evidence=defaultdict(lambda: defaultdict(int))
```